prac_05/emails.py

```diff
@@ -1,7 +1,30 @@
 
+QUIT = ""
+YES = "Y"
+DEFAULT_ACCEPT = ""
+
 
 def main():
-    print()
+    email_to_name = {}
+    user_email = input("Please enter a email to store:\n")
+    while user_email != QUIT:
+        name = get_name_from_email(user_email)
+        conformation = input("Is {} your name? (Y/N)\n".format(name)).upper()
+        if conformation != YES and conformation != DEFAULT_ACCEPT:
```

Comment:
See comment for line 3 as to why I think you may want to change '!= YES' to 'not in YES'.